File: pset6/mario/more/mario.py
# import get_int from cs50 library
from cs50 import get_int

# get input between 0 and 23
while True:
    h = get_int("Height: ")
    if h > 0 and h <= 8:
        break
# prints the pattern
for i in range(h):
    # first loop for spaces
    for j in range(1, h - i):
        print(" ", end='')
    # second loop for #
    for k in range(i + 1):
        print("#", end='')
    # print space between pyramids
    print("  ", end='')
    # second pyramid
    for m in range(i + 1):
        print("#", end='')
    print()

File: pset1/mario/more/mario.c
#include <cs50.h>
#include <stdio.h>
#include <stdlib.h>

int main(void)
{
    int h;
    //prompt user for height of pyramid from 0 to 23
    do
    {
        h = get_int("Height: ");
    }
    while (h < 0 || h  > 23);
// main loop for height of pyramid
    for (int i = 0; i < h; i++)
    {
        //1st loop for spaces before #
        for (int j = h - i; j > 1; j--)
        {
            printf(" ");
        }
        //2nd nested loop for 1st pyramid
        for (int k = 0; k < i + 1; k++)
        {
            printf("#");
        }
        //use printf instead of loop to print spaces between pyramids
        printf("  ");
        //4th loop fro second pyramid
        for (int m = 0; m < i + 1; m++)
        {
            printf("#");
        }
        //puts new line after every loop
        printf("\n");
    }


}